### PR TITLE
Add data source for forwarding rule

### PR DESCRIPTION
**Pull request by jphalip**

### CODE REVIEW

File: google/data_source_google_compute_forwarding_rule.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+			"ip_protocol": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"load_balancing_scheme": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```
[the remark this answers is not in the document]
Comment (jphalip) [reply google/data_source_google_compute_forwarding_rule.go]: Which parameters are you exactly referring to? Note: Here I've reused the same schema as in `resource_compute_forwarding_rule.go`.

[the remark this answers is not in the document]
Comment (ndmckinley) [reply google/data_source_google_compute_forwarding_rule.go]: Right, but as this is a data source, fundamentally all the parameters except the name, region, and project are Computed - no changes are made based on the user setting them.

File: google/data_source_google_compute_forwarding_rule.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	d.Set("name", frule.Name)
+	d.Set("port_range", frule.PortRange)
+	d.Set("ports", frule.Ports)
+	d.Set("subnetwork", frule.Subnetwork)
+	d.Set("network", frule.Network)
+	d.Set("target", frule.Target)
+	d.Set("project", project)
+	d.Set("region", region)
+
+	d.SetId(frule.Name)
```
[the remark this answers is not in the document]
Comment (jphalip) [reply google/data_source_google_compute_forwarding_rule.go]: Just pushed an update for that. Does that work?

File: google/data_source_google_compute_forwarding_rule.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jphalip) [reply google/data_source_google_compute_forwarding_rule.go]: Ok, so just to be clear, are you saying that this should be the schema? https://gist.github.com/jphalip/c0409e5b3e898898ce7706eb48b83950

[the remark this answers is not in the document]
Comment (ndmckinley) [reply google/data_source_google_compute_forwarding_rule.go]: That looks right to me, yes - I think the tests should pass with that schema, and that accurately represents what this does.

**Pull request merged**